Gemfile

```diff
@@ -42,6 +42,10 @@ group :bench do
 end
 
 group :test do
+  # allows for rspec-like invocation of single tests
+  # e.g.: $ m test/path.rb:lineNum
+  gem 'm', '~> 1.4.2'
```

Comment:
this is super handy!!! when I was testing the deserialization stuff, I just ran:

```
m test/action_controller/json_api/deserialization_test.rb:88
```